components/collector/src/source_collectors/dependency_track/base.py

```diff
@@ -24,7 +24,7 @@ class DependencyTrackProject(TypedDict):
     lastBomImport: int
     name: str
     uuid: str
-    version: str
+    version: NotRequired[str]
```

Comment:
Does it not work to set `version: str = "unknown"` ?

Reply:
No, `TypedDict` does not allow for setting default values.